py3dtilers/Common/lod_node.py

```diff
@@ -18,6 +20,8 @@ class LoaNode(GeometryNode):
     The LoaNode also takes a dictionary stocking the indexes of the features contained in each polygon.
     """
 
-    def __init__(self, features_node, geometric_error=50, polygons=list()):
+    DEFAULT_GEOMETRIC_ERROR = 20
```

Comment:
Stick to the practice of having one class defined per file (named after the class). Thus split lod_node.py in two files.